components.css

```diff
@@ -0,0 +1,327 @@
+@import url('../Components/Avatar/avatar.css');
+@import url('../Components/Alert/alert.css');
+@import url('../Components/Buttons/buttons.css');
+@import url('../Components/Images/images.css');
+@import url('../Components/Lists/list.css');
+@import url('../Components/Model/model.css');
+@import url('../Components/Badge/badge.css');
+@import url('../Components/Cards/cards.css');
+@import url('../NavbarLeft/navbar.css');
+@import url('../MediaQueries/colorchange.css');
+@import url('../Header/componentsHeader.css');
+@import url('../Colors/rootColors.css');
+@import url('../Icons/icons.css');
+
+body {
+    margin: 0;
+    padding: 0;
+}
+
+main {
+    margin-left: 5rem;
+    padding: 1rem;
+}
+
+/* Scroll-Bar */
+body::-webkit-scrollbar {
```

Comment:
My mouse scroll wheel is not working :D so custom scrollbar, look great, but they hamper a11y a lot.
In fact, mozilla team does not approve use of custom scrollbars. 
Learn more : https://adrianroselli.com/2019/01/baseline-rules-for-scrollbar-usability.html
Tips to make it great again:
- Increase width of scrollbar
- Change color to lightest (here) as it's tough to distinguish it from page. 
- If your design allows, remove it